runtime/opt/taupage/init.d/02-register-td-agent.py

```diff
@@ -81,11 +81,11 @@ def update_configuration_from_template():
     fluentd_customlog_filter_exclude = logging_config.get('customlog_filter_exclude', None)
     fluentd_loglevel = logging_config.get('fluentd_loglevel', 'info')
     fluentd_s3_raw_log_format = logging_config.get('s3_raw_log_format', 'true')
-    fluentd_s3_region = logging_config.get('s3_region', 'eu-central-1')
-    fluentd_s3_bucket = logging_config.get('s3_bucket')
+    fluentd_s3_region = logging_config.get('s3_region', aws_region)
+    fluentd_s3_bucket = logging_config.get('s3_bucket', 'zalando-logging-'+aws_account+'-'+aws_region)
     fluentd_s3_timekey = logging_config.get('s3_timekey', '5m')
     fluentd_s3_acl = logging_config.get('s3_acl', 'bucket-owner-full-control')
-    fluentd_rsyslog_host = logging_config.get('rsyslog_host')
+    fluentd_rsyslog_host = logging_config.get('rsyslog_host', None)
```

Comment:
Doesn't `dict.get()` return `None` by default anyway

Reply:
yes it does.